app/components/viewer/ReplayCars.tsx

```diff
@@ -0,0 +1,254 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable import/prefer-default-export */
+import {
+    extend, useFrame, Canvas, useThree, useLoader,
+} from '@react-three/fiber';
+import * as THREE from 'three';
+import {
+    BufferAttribute, BufferGeometry, DoubleSide, MeshPhongMaterial, Triangle,
+} from 'three';
+import React, {
+    useRef, useState, useEffect, useMemo, Suspense,
+} from 'react';
+import { Text } from 'troika-three-text';
+import { GLTFLoader } from 'three-stdlib';
+import { useFBX, useGLTF } from '@react-three/drei';
+import { ReplayData } from '../../lib/api/apiRequests';
+import fonts from '../../assets/fonts';
+import { ReplayDataPoint } from '../../lib/replays/replayData';
+import { InputOverlay } from './InputOverlay';
+import { getRaceTimeStr } from '../../lib/utils/time';
+import vecToQuat from '../../lib/utils/math';
+
+extend({ Text });
+
+interface ReplayCarProps {
+    replay: ReplayData;
+    timeLineGlobal: any;
+    camera: any;
+    orbitControlsRef: any;
+    showInputOverlay: boolean;
+    fbx: THREE.Object3D;
+    replayCarOpacity: number;
+    cameraMode: string;
+}
+
+const ReplayCar = ({
+    replay, timeLineGlobal, camera, orbitControlsRef, showInputOverlay, fbx, replayCarOpacity, cameraMode,
+}: ReplayCarProps) => {
+    const mesh = useRef<THREE.Mesh>(null!);
+    const meshTxt = useRef<THREE.Mesh>(null!);
+    const stadiumCarMesh = useRef<THREE.Mesh>(null!);
+    const velocityRef = useRef<THREE.Mesh>(null!);
+
+    const CurrentSampleRef = useRef<ReplayDataPoint>(null!);
+
+    let sampleIndex = 0;
+    let hovered: boolean = false;
+    let prevOnClick : number = Date.now();
+
+    const pressedKeys: any = {};
+
+    const childrend: THREE.Mesh = fbx.children[0] as THREE.Mesh;
```

Comment:
`childrend` -> `children`. Also, it's not children, its the car mesh, so I would call it something like `carMesh`. And why not directly input a Mesh as a prop, instead of the full fbx thingy. This will error if the .children array is empty for some reason.